test/unit/runner.spec.js

```diff
@@ -453,7 +453,63 @@ describe('Runner', function() {
         newRunner.runTest();
       }
       expect(fail, 'to throw', 'allow unhandled errors');
-      done();
+    });
+
+    it('should not allow unhandled errors in hooks to propagate through', function() {
+      suite.beforeEach(function() {
+        throw new Error('this error will not propagate');
+      });
+      var runner = new Runner(suite);
+      expect(function() {
+        runner.hook('beforeEach', function() {});
+      }, 'not to throw');
+    });
```

Comment:
I don't trust this test, since `expect(...)` passes even with `runner.allowUncaught = true;`. First it passes, then the test fails. Please use the same logic as of the following two tests.
To be consistent with the existing code, use `noop` instead of `function() {}`.

Reply:
Strange, I saw the test fail reliably when I added `runner.allowUncaught = true;`. Anyway, the pattern used in the other two tests is definitely 100% reliable and would provide more consistency.